bundler/lib/bundler/templates/newgem/newgem.gemspec.tt

```diff
@@ -41,12 +37,6 @@ Gem::Specification.new do |spec|
   spec.extensions = ["ext/<%= config[:underscored_name] %>/extconf.rb"]
 <%- end -%>
 
-  # Uncomment to register a new dependency of your gem
-  # spec.add_dependency "example-gem", "~> 1.0"
```

Comment:
Please restore these lines. They are not related Rust gem.

Reply:
You can maybe move it to an `else` block, since it the case of `config[:ext] == 'rust'`, it should be unnecessary to explain how to add a dependency?

Reply:
It should be....but it's nice to do. :) I'll restore it.